tests/integration/create_get_delete/integration_test.go

```diff
@@ -0,0 +1,200 @@
+	region        = "us-west-2"
+)
+
+var (
+	eksctlPath string
+
+	// Flags to help with the development of the integration tests
+	clusterName    string
+	doCreate       bool
+	doDelete       bool
+	kubeconfigPath string
+
+	kubeconfigTemp bool
+)
+
+func TestCreateIntegration(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Integration - Create Suite")
+}
+
+type tInterface interface {
+	GinkgoTInterface
+	Helper()
+}
+
+type tHelper struct{ GinkgoTInterface }
+
+func (t *tHelper) Helper()      { return }
+func (t *tHelper) Name() string { return "eksctl-test" }
+
+func newKubeTest() (*harness.Test, error) {
+	t := &tHelper{GinkgoT()}
+	l := &logger.TestLogger{}
+	h := harness.New(harness.Options{Logger: l.ForTest(t)})
+	if err := h.Setup(); err != nil {
+		return nil, err
+	}
+	if err := h.SetKubeconfig(kubeconfigPath); err != nil {
+		return nil, err
+	}
+	return h.NewTest(t), nil
+}
+
+var _ = Describe("Create (Integration)", func() {
+
+	BeforeSuite(func() {
+		kubeconfigTemp = false
+		if kubeconfigPath == "" {
+			wd, _ := os.Getwd()
+			f, _ := ioutil.TempFile(wd, "kubeconfig-")
+			kubeconfigPath = f.Name()
+			kubeconfigTemp = true
+		}
+	})
+
+	AfterSuite(func() {
+		gexec.KillAndWait()
+		if kubeconfigTemp {
+			os.Remove(kubeconfigPath)
+		}
+		if doCreate && doDelete {
+			integration.CleanupAws(clusterName, region)
+		}
+	})
+
+	Describe("when creating a cluster with 1 node", func() {
+		var (
+			err     error
+			session *gexec.Session
+		)
+
+		It("should not return an error", func() {
+			if !doCreate {
+				fmt.Fprintf(GinkgoWriter, "will use existing cluster %s", clusterName)
+				return
+			}
+
+			fmt.Fprintf(GinkgoWriter, "Using kubeconfig: %s\n", kubeconfigPath)
+
+			if clusterName == "" {
+				clusterName = utils.ClusterName("", "")
+			}
+
+			args := []string{"create", "cluster", "--name", clusterName, "--node-type", "t2.medium", "--nodes", "1", "--region", region, "--kubeconfig", kubeconfigPath}
+
+			command := exec.Command(eksctlPath, args...)
+			session, err = gexec.Start(command, GinkgoWriter, GinkgoWriter)
+
+			if err != nil {
+				Fail(fmt.Sprintf("error starting process: %v", err), 1)
+			}
+
+			session.Wait(createTimeout * time.Minute)
+			Expect(session.ExitCode()).Should(Equal(0))
+		})
+
+		It("should have created an EKS cluster", func() {
+			session := aws.NewSession(region)
+			Expect(session).To(HaveEksCluster(clusterName, awseks.ClusterStatusActive, "1.10"))
+		})
+
+		It("should have the required cloudformation stacks", func() {
+			session := aws.NewSession(region)
+
+			Expect(session).To(HaveCfnStack(fmt.Sprintf("EKS-%s-VPC", clusterName)))
+			Expect(session).To(HaveCfnStack(fmt.Sprintf("EKS-%s-ControlPlane", clusterName)))
+			Expect(session).To(HaveCfnStack(fmt.Sprintf("EKS-%s-ServiceRole", clusterName)))
+			Expect(session).To(HaveCfnStack(fmt.Sprintf("EKS-%s-DefaultNodeGroup", clusterName)))
+		})
+
+		It("should have created a valid kubectl config file", func() {
```

Comment:
> It would be good if we could get the kubeconfig test also working (i ran out of time before holiday to look at the glog issues).

@richardcase I am not hitting a glog issue here, not sure why. But I've uncommented the test and it works! 🎉